### DISCUSSION TITLE
Fingerprint problem

### DISCUSSION THREAD

**Discussion by talkkkum** [phhusson/treble_experimentations#1645] [Q&A]
Hello, sir! It is a good project! But when I flash it, i can't use under screen fingerprints on this gsi. My phone is Redmi 10x 5g. I hope that you can fix this problem, thank u very much!!!

**Comment by DiogoCosta35** [comment]
AOSP 10.0 v222 fingerprint reader is not working on moto g7 play please fix this problem please
